### PR TITLE
WILL-1120 - 404 overview

### PR DESCRIPTION
**Pull request by MFlor**
Adding logging of 404 errors, also adding notifications for when a URL exceeds threshold.

### CODE REVIEW

File: src/Database/Migrations/CreateNotFoundTable.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+        $sql = "
+        SET sql_notes = 1;
+        CREATE TABLE `$table` (
+          `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+          `url` text CHARACTER SET utf8 NOT NULL,
+          `url_hash` char(32) COLLATE utf8mb4_unicode_520_ci NOT NULL,
+          `locale` varchar(2) CHARACTER SET utf8 NOT NULL,
+          `hits` int(11) unsigned NOT NULL DEFAULT 0,
+          `notification_sent` tinyint(1) unsigned NOT NULL DEFAULT 0,
```
Comment (alfhen) [inline src/Database/Migrations/CreateNotFoundTable.php]: Perhaps a timestamp `notified_at` rather than a tiny int? for debug purposes.

**Comment by alfhen** [comment]
LGTM 👍 😄

File: src/Database/Migrations/CreateNotFoundTable.php (hunk 1 above)
Comment (MFlor) [reply src/Database/Migrations/CreateNotFoundTable.php]: Det må blive i v1.1 😬

Comment (alfhen) [reply src/Database/Migrations/CreateNotFoundTable.php]: Fair enough 😉

**Pull request merged**